src/main/java/com/eatthepath/uuid/FastUUID.java

```diff
@@ -51,83 +76,82 @@ private FastUUID() {
      * {@link UUID#toString()}.
      *
      * @param uuidSequence the character sequence from which to parse a UUID
-     *
      * @return the UUID represented by the given character sequence
-     *
      * @throws IllegalArgumentException if the given character sequence does not conform to the string representation as
-     * described in {@link UUID#toString()}
+     *                                  described in {@link UUID#toString()}
      */
-    public static UUID parseUUID(final CharSequence uuidSequence) {
+    public static UUID parseUUID(final String uuidSequence) {
```

Comment:
It's actually reasonably important that this be a `CharSequence`. I don't see anything in here that strictly requires a `String` instance; am I missing something?